config/main.py

```diff
@@ -673,8 +685,12 @@ def _stop_services():
         pass
 
     click.echo("Stopping SONiC target ...")
-    clicommon.run_command("sudo systemctl stop sonic.target")
```

Comment:
>stop [](http://example.com/codeflow?start=42&length=4)

It we tried stop and failed, just report the failure. Stop all of them twice will not be a general solution since there is possibility of failing again.

Reply:
We can't report and exit out of a 'config reload' command. The system may be down and it is likely that it is not reachable. So we need to take a corrective action which is to perform stop operation on individual targets and ignore any failures. This is a general solution as the original stop operation and the stop_all operation are different in nature.

Reply:
Adding stop_all method will certainly reduce the possibility of failing again because a "sonic.target" stop job is very sensitive to the conflicting jobs for any of the dependent services. 

On the other hand stopping individual services would mean that the likelihood of race condition significantly reduces

Reply:
> "sonic.target" stop job is very sensitive to the conflicting jobs for any of the dependent services. 

This seems something we need to fix first.

Reply:
I'm not sure if this can be fixed. Well point to note here is that only "stop job" is sensitive. the start job isn't. The systemd documentation mentions about the startup behavior of a wants dependency. [Link](https://www.freedesktop.org/software/systemd/man/systemd.unit.html) but not of a stop behavior.